README.md

```diff
@@ -239,6 +239,8 @@ CSS          2 repos        █░░░░░░░░░░░░░░░░
 
 `IGNORED_REPOS`  flag can be set to `"waka-readme-stats, my-first-repo"` (just an example) to ignore some repos you don’t want to be counted
 
+`WAKATIME_API_URL` flag can be set if you are hosting your own wakatime compliant backend like [wakapi](https://github.com/muety/wakapi). You only have to supply your base url like so `https://wakapi.dev`
```

Comment:
Maybe we should shouw that this is an example link, changing it to something like `https://your-own-wakapi.dev`? Don't you think so?